src/librustc_mir/diagnostics.rs

```diff
@@ -2187,6 +2187,53 @@ fn main() {
 ```
 "##,
 
+E0713: r##"
+When you write `&expr` and `expr` is not a "path" rooted in some
```

Comment:
Make sure to check over the examples in that gist; don't just cut-and-paste their content blindly. In particular, the `fn bar`'s, starting on line 8, probably should be returning a `&i32`, not a `i32`.

(The code as written doesn't compile because it tries to dereference an `i32`, but that's not the error we are trying to illustrate here.)

Reply:
Note that, unless the example is tagged with ignore, we actually test this.

Reply:
(True; not everyone runs the full test suite locally, but the error would have been caught.)